### PR TITLE
Update patient support query AB#15481

### PR DESCRIPTION
**Pull request by MikeLyttle**
# Implements [AB#15481](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15481)

## Description

- updates support page table to display entire HDID
- returns no result instead of 404 in patient support query
- tidies up functional tests to use shared functions

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## UI Changes

localhost-2023-05-01-164111

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by mk08sh — COMMENTED** [review]
✅

**Review by rayking29 — DISMISSED** [review]

File: Apps/Admin/Server/Services/SupportService.cs [hunk 1]
```diff
@@ -160,8 +160,15 @@ private async Task<PatientSupportDetails> GetPatientSupportDetailsAsync(UserProf
...
                 : new PatientDetailsQuery(Phn: queryString);
 
-            PatientQueryResult result = await this.patientRepository.Query(query, ct).ConfigureAwait(true);
-            return result.Items.SingleOrDefault();
+            try
+            {
+                PatientQueryResult result = await this.patientRepository.Query(query, ct).ConfigureAwait(true);
+                return result.Items.SingleOrDefault();
+            }
+            catch (ProblemDetailsException)
```
Comment (ytqsl) [inline Apps/Admin/Server/Services/SupportService.cs:168]: You may want to be more specific here eventually

**Review by ytqsl — APPROVED** [review]

File: Apps/Admin/Server/Services/SupportService.cs (hunk 1 above)
Comment (BronzBierd) [reply Apps/Admin/Server/Services/SupportService.cs:168]: Do we not have a value that can determine if its a "not found" error vs others?

Comment (MikeLyttle) [reply Apps/Admin/Server/Services/SupportService.cs]: Fixed.

**Review by ytqsl — APPROVED** [review]

**Review by rayking29 — APPROVED** [review]

**Review by BronzBierd — APPROVED** [review]
Looks good 👍

**Pull request merged**